dex-auth/dex-crds/base/kustomization.yaml

```diff
@@ -78,7 +78,3 @@ vars:
     fieldpath: data.application_secret
 configurations:
 - params.yaml
-images:
```

Comment:
@WillBeebe Its recommended that we retain the 'images' field in kustomization.yaml. Could you please retain this field and update it with the image name.
This allows us to parameterize images through CLI.

Reply:
No problem, that makes sense. :) Just pushed the change.